packages/waku/src/lib/plugins/vite-plugin-rsc-serve.ts

```diff
@@ -44,6 +50,17 @@ export function rscServePlugin(opts: {
             'Unsupported: build.rollupOptions.external is not an array',
           );
         }
+      } else if (opts.serve === 'partykit') {
```

Comment:
I think this can be merged with the previous condition. I'll make a follow-up PR later.